lua/flatbuffers/builder.lua

```diff
@@ -1,6 +1,7 @@
 local N = require("flatbuffers.numTypes")
 local ba = require("flatbuffers.binaryarray")
 local compat = require("flatbuffers.compat")
+local sunpack = compat.sunpack
```

Comment:
I just see "sun" pack when I read this :) Could we name stringunpack or something else?